addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/internal/data/Thermostat.java

```diff
@@ -45,11 +45,11 @@
     @SerializedName("target_temperature_low_c")
```

Comment:
Was there a reason why the binding only keeps track of Celsius not the Fahrenheit? According to the API there are a lot of Fahrenheit properties. eg:

```
"target_temperature_f": 72,
"target_temperature_c": 21.5,
"target_temperature_high_f": 80,
"target_temperature_high_c": 24.5,
"target_temperature_low_f": 65,
"target_temperature_low_c": 19.5,
"eco_temperature_high_f": 80,
"eco_temperature_high_c": 24.5,
"eco_temperature_low_f": 65,
"eco_temperature_low_c": 19.5,
"away_temperature_high_f": 80,
"away_temperature_high_c": 24.5,
"away_temperature_low_f": 65,
"away_temperature_low_c": 19.5,
```
Via commands it is also only possible to set Celsius temperatures. Should US citizens do their own conversions? Or is the idea that the future ESH unit conversion feature will solve this?

Reply:
Imho it's okay now or maybe even better to not spent time on adding the option for fahrenheit because the unit conversion should indeed resolve this instead.

Reply:
I do personally think it is better for all the bindings to use one scale, ie: celcius, and then do the conversions on the edge to what the user is seeing. You don't run into the issues of the scale not matching for some reasson and making an incorrect decision based on that.